WordPress/Classes/Services/BlogService.m

```diff
@@ -108,12 +124,15 @@ - (Blog *)lastUsedBlog
         return nil;
     }
 
-    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:@"Blog"];
-    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"visible = YES AND url = %@", url];
+    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:BlogEntity];
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%@ AND url = %@", @"visible = YES", url];
```

Comment:
Hmm... `NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%@ AND url = %@", @"visible = YES", url];` is what I'm seeing now. Probably no need to have `visible = YES` as a param.  Maybe just inline it all as it was previously? `NSPredicate *predicate = [NSPredicate predicateWithFormat:@"visible = YES AND url = %@", url];`